src/test/java/uk/gov/companieshouse/chipsrestinterfacesconsumer/service/impl/MessageProcessorServiceImplTest.java

```diff
@@ -27,27 +28,34 @@
     private ChipsRestClient chipsRestClient;
 
     @Mock
-    private MessageProducer retryMessageProducer;
+    private MessageProducer messageProducer;
 
     @Mock
     private ApplicationLogger logger;
 
     @InjectMocks
     private MessageProcessorServiceImpl messageProcessorService;
 
+    @BeforeEach
+    void Setup(){
+        ReflectionTestUtils.setField(messageProcessorService, "maxRetryAttempts", 10);
```

Comment:
the strings "chips-rest-interfaces-send-retry" and "chips-rest-interfaces-send-error" are duplicated lots of times, they can be replaced with constants

Reply:
Refractored